replication-api/src/main/java/com/connexta/replication/api/impl/persistence/DataManager.java

```diff
@@ -11,9 +11,9 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package com.connexta.ion.replication.api.persistence;
+package com.connexta.replication.api.impl.persistence;
```

Comment:
Why did we add the impl to the package name? This is still an interface in the api module.

Reply:
oops!

Reply:
I think IntelliJ played a joke on me